etc/conda3_packages-linux-64.txt

```diff
@@ -1,87 +1,95 @@
+  - libstdcxx-ng=8.2.0=hdf63c60_1
+  - libuuid=1.0.3=h1bed415_2
   - libxcb=1.13=h1bed415_1
-  - libxml2=2.9.8=h26e45fe_1
-  - matplotlib=2.2.2=py36h0e671d2_1
-  - more-itertools=4.2.0=py36_0
-  - ncurses=6.1=hf484d3e_0
+  - libxml2=2.9.9=he19cac6_0
+  - lzo=2.10=h49e0be7_2
+  - matplotlib=3.0.2=py37h5429711_0
+  - more-itertools=5.0.0=py37_0
+  - ncurses=6.1=he6710b0_1
   - nomkl=3.0=0
-  - numexpr=2.6.5=py36_nomklhaa809a4_0
-  - numpy=1.14.5=py36h28100ab_3
-  - numpy-base=1.14.5=py36h2b20989_3
-  - openssl=1.0.2o=h20670df_0
-  - pandas=0.23.1=py36h637b7d7_0
+  - numexpr=2.6.9=py37h2ffa06c_0
+  - numpy=1.15.4=py37h99e49ec_0
+  - numpy-base=1.15.4=py37h2f8d375_0
+  - openssl=1.1.1b=h7b6447c_0
+  - pandas=0.24.1=py37he6710b0_0
   - pcre=8.42=h439df22_0
-  - pip=10.0.1=py36_0
-  - pluggy=0.6.0=py36hb689045_0
-  - psutil=5.4.6=py36h14c3975_0
-  - py=1.5.3=py36_0
-  - pycparser=2.18=py36hf9f622e_1
-  - pyopenssl=18.0.0=py36_0
-  - pyparsing=2.2.0=py36hee85983_1
-  - pyqt=5.9.2=py36h751905a_0
-  - pysocks=1.6.8=py36_0
-  - pytest=3.6.2=py36_0
-  - pytest-arraydiff=0.2=py36_0
-  - pytest-astropy=0.4.0=py36_0
-  - pytest-doctestplus=0.1.3=py36_0
-  - pytest-openfiles=0.3.0=py36_0
-  - pytest-remotedata=0.3.0=py36_0
-  - python=3.6.6=hc3d631a_0
-  - python-dateutil=2.7.3=py36_0
-  - pytz=2018.5=py36_0
-  - pyyaml=3.12=py36hafb9ca4_1
-  - qt=5.9.5=h7e424d6_0
-  - readline=7.0=ha6073c6_4
-  - requests=2.19.1=py36_0
-  - scikit-learn=0.19.1=py36_nomklh27f7947_0
-  - scipy=1.1.0=py36_nomklh9d22d0a_0
-  - setuptools=39.2.0=py36_0
-  - sip=4.19.8=py36hf484d3e_0
-  - six=1.11.0=py36h372c433_1
-  - sqlalchemy=1.2.8=py36h14c3975_0
-  - sqlite=3.24.0=h84994c4_0
-  - tk=8.6.7=hc745277_3
-  - tornado=5.0.2=py36_0
-  - urllib3=1.23=py36_0
-  - wheel=0.31.1=py36_0
+  - pip=19.0.3=py37_0
+  - pluggy=0.9.0=py37_0
+  - psutil=5.5.0=py37h7b6447c_0
+  - py=1.8.0=py37_0
+  - pycparser=2.19=py37_0
+  - pyopenssl=19.0.0=py37_0
+  - pyparsing=2.3.1=py37_0
+  - pyqt=5.9.2=py37h05f1152_2
+  - pysocks=1.6.8=py37_0
+  - pytables=3.4.4=py37h71ec239_0
+  - pytest=4.3.0=py37_0
+  - pytest-arraydiff=0.3=py37h39e3cac_0
+  - pytest-astropy=0.5.0=py37_0
+  - pytest-doctestplus=0.2.0=py37_0
+  - pytest-openfiles=0.3.2=py37_0
+  - pytest-remotedata=0.3.1=py37_0
+  - python=3.7.2=h0371630_0
+  - python-dateutil=2.8.0=py37_0
+  - pytz=2018.9=py37_0
+  - pyyaml=3.13=py37h14c3975_0
+  - qt=5.9.7=h5867ecd_1
+  - readline=7.0=h7b6447c_5
+  - requests=2.21.0=py37_0
+  - scikit-learn=0.20.2=py37h22eb022_0
+  - scipy=1.2.1=py37he2b7bc3_0
+  - setuptools=40.8.0=py37_0
+  - sip=4.19.8=py37hf484d3e_0
+  - six=1.12.0=py37_0
+  - snappy=1.1.7=hbae5bb6_3
+  - sqlalchemy=1.2.18=py37h7b6447c_0
+  - sqlite=3.26.0=h7b6447c_0
+  - tk=8.6.8=hbc83047_0
+  - tornado=5.1.1=py37h7b6447c_0
+  - urllib3=1.24.1=py37_0
+  - wheel=0.33.1=py37_0
   - xz=5.2.4=h14c3975_4
   - yaml=0.1.7=had09818_2
-  - zlib=1.2.11=ha838bed_2
+  - zlib=1.2.11=h7b6447c_3
   - pip:
     - deprecated==1.2.4
+    - tables==3.4.4
```

Comment:
Why is this a `pip install` rather than using the `pytables` conda package?  I don't think we want to be using pip when a conda package exists since that may cause some difficulties with version management of the related numpy packages.

Reply:
Right. My error, I have also to add pytables to the bleed files.